timely/src/progress/subgraph.rs

```diff
@@ -689,17 +690,7 @@ impl<T: Timestamp> PerOperatorState<T> {
             }
 
             incomplete
-        }
-        else {
-
-            // If the operator is closed and we are reporting progress at it, something has surely gone wrong.
-            if self.shared_progress.borrow_mut().frontiers.iter_mut().any(|x| !x.is_empty()) {
-                println!("Operator prematurely shut down: {}", self.name);
-                println!("  {:?}", self.notify);
-                println!("  {:?}", self.shared_progress.borrow_mut().frontiers);
-                panic!();
-            }
-
```

Comment:
Does this still need to be deleted with the proxy `EmptyOperator`?

Reply:
We still check it, it is just moved further up https://github.com/TimelyDataflow/timely-dataflow/pull/441/files#diff-ac6e819a79d1d2170b82aee757936570e29d586f825d119f8b1092ee8df187f4R474